selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/ImportAndValidateEclipseCheProjectTest.java

```diff
@@ -180,9 +170,31 @@ public void checkErrorMarkersInEditor() {
     editor.waitMarkerInvisibility(ERROR, 12);
 
     // check error marker in the ts file
-    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_TS_FILE));
+    projectExplorer.openItemByPath(PATH_TO_TS_FILE);
     editor.waitActive();
     editor.typeTextIntoEditor("q");
     editor.waitMarkerInPosition(ERROR, 1);
   }
+
+  private void closeErrorDialog() {
+    // if the error dialog is appeared more 5 times, the test will be failed
+    int counterErrorDialog = 0;
+
+    while (informationDialog.isFormOpened()) {
+      counterErrorDialog++;
+
+      if (counterErrorDialog > 5) {
+
+        Assert.assertEquals(
```

Comment:
"assertEquals" looks redundant, in my opinion, "Assert.fail()" is better to use in this case:
```
fail("The unexpected error information dialog is appeared more than "+ counterErrorDialog +" times");
```

Reply:
OK